Packs/CommonScripts/Scripts/GetErrorsFromEntry/GetErrorsFromEntry.py

```diff
@@ -30,9 +30,24 @@ def main():
         args = demisto.args()
         # the entry_id argument can be a list of entry ids or a single entry id
         entry_ids = args.get('entry_id', demisto.get(demisto.context(), 'lastCompletedTaskEntries'))
+
         entry_ids = argToList(entry_ids)
+        entries = []
+
+        if is_xsiam_or_xsoar_saas():
+            try:
+                entry_ids_str = ",".join(entry_ids)
+                entries = demisto.executeCommand('getEntriesByIDs', {'entryIDs': entry_ids_str})
+            except ValueError as e:
+                if "Unsupported Command" not in str(e):
+                    raise e
```

Comment:
don't we want to get back to the old behavior in case of failure and just print to demisto.error

Reply:
we want to proceed with the old behavior only when `getEntriesByIDs` fails on  "Unsupported Command", otherwise we want to raise the exception.